File: 1020.go
package main

import "fmt"

var ageInDays int

func main() {
	fmt.Scanf("%d", &ageInDays)
	years := ageInDays / 365
	ageInDays %= 365
	months := ageInDays / 30
	fmt.Printf("%d ano(s)\n%d mes(es)\n%d dia(s)\n", years, months, ageInDays%30)
}

File: c++/uri-online-judge/1020.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  int ageInDays; cin >> ageInDays;

  int years  = ageInDays / 365;
  ageInDays %= 365;
  int months = ageInDays / 30;
  int days   = ageInDays % 30;

  cout << years << " ano(s)" << endl;
  cout << months << " mes(es)" << endl;
  cout << days << " dia(s)" << endl;
  return 0;
}